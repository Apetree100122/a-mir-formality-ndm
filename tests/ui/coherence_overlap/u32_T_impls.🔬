[
    crate core {
        trait Foo<> where [] {}
        impl<> Foo<> for u32 where [] {}
        impl<ty T> Foo<> for T where [] {}
    }
]
