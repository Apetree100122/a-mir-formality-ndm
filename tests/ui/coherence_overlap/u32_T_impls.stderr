Error: impls may overlap: `impl <> Foo < > for (rigid (scalar u32)) where [] { }` vs `impl <ty> Foo < > for ^ty0_0 where [] { }`
